frontend/babel.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (api) => {
```

Comment:
This might improve DX.
https://mui.com/guides/minimizing-bundle-size/#development-environment